src/race.js

```diff
@@ -0,0 +1,83 @@
+import Car from "./car.js";
+
+let cars = [];
+let winnerArray = [];
+const RACE_CONDITION_NUMBER = 4;
+const COMMA = ', ';
+const COMMA_BLANK = 2;
```

Comment:
이게 맞는건지는 모르겠는데, 일단 참고만 하는게 좋을 것 같아서 남겨봅니다 :)

이런 고정된 상수 변수를 모아두어서 모듈화 시키는것도 좋을것 같아요.

```suggestion
const constantValues = {
  RACE_CONDITION_NUMBER = 4,
  COMMA = ', ',
  COMMA_BLANK = 2
}
```
이렇게 만들고, 또 `alert` 메세지들도 모으는 게 좋을 것 같아요.

```javascript
const message = {
CAR_NAME_5_LETTERS_LIMIT : '자동차 이름의 길이를 5자 이하로 해주세요.',
...
}
```

이런 고정된 친구들을 모아서 다른폴더에 넣고, 이를 다시 `import` 하는게 어떨까요?
```javascript
import {conatntValues, message} from "../constants.js";

...

if(carNameLength > 5) {
  alert(message.CAR_NAME_5_LETTERS_LIMIT);
}
```

이런식으로 정리해도 보기에 깔끔할 것 같기도 합니당! 

그냥 제 의견이긴합니다.. ㅎㅎ

Reply:
와우...! 이렇게 하면 상수나 msg를 한 번에 관리할 수 있겠네요 
감사합니닷 :)